src/data_conn.c

```diff
@@ -626,6 +649,10 @@ replica_thread(void *arg)
 	ev.data.ptr = NULL;
 
 	MTX_LOCK(&r->r_mtx);
+	r->data_eventfd = r_data_eventfd;
```

Comment:
assign these vars after epoll_ctl of iofd is successful

Reply:
ok